quicktravel_client.gemspec

```diff
@@ -21,6 +21,8 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'activesupport', '>= 2.0.0'
   spec.add_dependency 'facets'
   spec.add_dependency 'money', '>= 3.0', '< 6.0' # 6.0 starts to deprecate/split
+  #this dependency cannot be found when loaded this way for some reason...
```

Comment:
Do we need these 2 lines?

Reply:
Well this should be in here I just couldn't get it to work, as it is right now all projects that use this gem have to also load this dependency otherwise the gem is broken

Reply:
I'm happy to accept a merge on the basis this will be looked at after. I don't wish to hold back a PR over an existing issue.

Having said that, I think a few hours (but hopefully just a few minutes) should hopefully solve this issue whilst we remember it.